lib/lita/handlers/reviewme.rb

```diff
@@ -136,6 +137,17 @@ def get_room(response)
         Lita::Room.find_by_id(room_id) || Lita::Room.new(room_id)
       end
 
+      def all_reviewers(room)
+        ns_redis(room.id).lrange(REDIS_LIST, 0, -1)
+      end
+
+      def notify(reviewer, url)
+        return unless reviewer.chat_handle
+
+        target = Lita::Source.new(user: reviewer.chat_handle)
+        robot.send_message(target, "Hi there, you've been assigned to review #{url}.")
```

Comment:
Turns out this doesn't work they way I thought. Will require a little more work.